.env

```diff
@@ -0,0 +1,12 @@
+POSTGRES_DB=keycloak_db
+POSTGRES_USER=keycloak_db_user
+POSTGRES_PASSWORD=keycloak_db_user_password
+KEYCLOAK_ADMIN=admin
+KEYCLOAK_ADMIN_PASSWORD=password
+KEYCLOAK_CLIENT_ID="nextjs"
+KEYCLOAK_CLIENT_SECRET="gX0nfIp57bO3bY68br3kdTXung2Auwpr"
+KEYCLOAK_ISSUER="http://localhost:8080/realms/shlp"
+NEXTAUTH_URL="http://localhost:3000"
+NEXTAUTH_SECRET="ArmMS4LBmH3VTR77UrSAY2lPs04HO0Nk2/4BcU0jMvc="
+NEXT_CONTAINER_KEYCLOAK_ENDPOINT="http://keycloak:8080"
+NEXT_LOCAL_KEYCLOAK_URL="http://localhost:8080"
```

Comment:
Lets move this `.env` data into the `.env.example`. and also copy the env keys into the `.env.production.example`. 

We separate these to have environment specific config defined. 

The `.env` isnt being committed to the repo, we make a copy of the example and populate the details needed per instance